Paper.tex

```diff
@@ -1661,6 +1661,7 @@ \section{Signing Transactions}\label{app:signing}
 We assume the existence of functions $\mathtt{\small ECDSAPUBKEY}$, $\mathtt{\small ECDSASIGN}$ and $\mathtt{\small ECDSARECOVER}$. These are formally defined in the literature, \eg by \cite{ECDSAcerticom}.
 \begin{eqnarray}
 \mathtt{\small ECDSAPUBKEY}(p_{\mathrm{r}} \in \mathbb{B}_{32}) & \equiv & p_{\mathrm{u}} \in \mathbb{B}_{64} \\
+\ e & \equiv & \hyperlink{h_of_T}{h(T)}\\
```

Comment:
I think `e` should be explained in the next paragraph (which explains p_r, p_u and v).

Reply:
Done.